packages/trip-form/src/TripOptions/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement, useState } from "react";
+
+import GeneralSettingsPanel from "../GeneralSettingsPanel";
+
+import FeaturedOption from "./FeaturedOption";
+import FeaturedOptionOverlay from "./FeaturedOptionOverlay";
+import ModeRow from "./ModeRow";
+import TransitOptions from "./TransitOptions";
+import { Company, QueryProps } from "./types";
+import * as S from "./styled";
+
+interface ComponentProps {
+  footer?: ReactElement;
+  supportedCompanies: Company[];
```

Comment:
Add JSDocs for these props so they appear in storybook docs for that component story.

Reply:
Added in 6d0d62dd6cd2cba86e6fab24c6c8d7222ee0e239, but @landonreed could you possibly expand since I'm not entirely sure what some of them do